apps/visualizer/src/pages/home.tsx

```diff
@@ -10,31 +10,51 @@ export default function Home() {
 
   const [errorMessage, setError] = createSignal<string | null>(null)
 
-  async function handleDrop(e: DragEvent) {
+  function handleDrop(e: DragEvent) {
     e.preventDefault()
+    processFile(e.dataTransfer?.files || new FileList())
```

Comment:
I'm curious what is the difference between `FileList` and `[]` krub

Reply:
The type of `[]` is `undefined[]`, which might raise a Type incompatible error.